### ISSUE TITLE
Error getting snapshot: 422

### ISSUE THREAD

**Issue by thomas-hass** [ljmerza/frigate_plate_recognizer#63]
`2024-11-04 14:32:53,548 - __main__ - DEBUG - mqtt message: {'before': {'id': '1730727171.175817-t6tyxn', 'camera': 'tapo_innkjorsel', 'frame_time': 1730727171.684443, 'snapshot': {'frame_time': 1730727171.684443, 'box': [16, 34, 114, 122], 'area': 8624, 'region': [0, 0, 320, 320], 'score': 0.8046875, 'attributes': []}, 'label': 'car', 'sub_label': None, 'top_score': 0.75390625, 'false_positive': False, 'start_time': 1730727171.175817, 'end_time': None, 'score': 0.8046875, 'box': [16, 34, 114, 122], 'area': 8624, 'ratio': 1.1136363636363635, 'region': [0, 0, 320, 320], 'active': True, 'stationary': False, 'motionless_count': 0, 'position_changes': 1, 'current_zones': [], 'entered_zones': [], 'has_clip': False, 'has_snapshot': True, 'attributes': {}, 'current_attributes': [], 'pending_loitering': False}, 'after': {'id': '1730727171.175817-t6tyxn', 'camera': 'tapo_innkjorsel', 'frame_time': 1730727171.773814, 'snapshot': {'frame_time': 1730727171.684443, 'box': [16, 34, 114, 122], 'area': 8624, 'region': [0, 0, 320, 320], 'score': 0.8046875, 'attributes': []}, 'label': 'car', 'sub_label': None, 'top_score': 0.7578125, 'false_positive': False, 'start_time': 1730727171.175817, 'end_time': 1730727173.165905, 'score': 0.78125, 'box': [0, 52, 92, 160], 'area': 9936, 'ratio': 0.8518518518518519, 'region': [0, 0, 320, 320], 'active': True, 'stationary': False, 'motionless_count': 1, 'position_changes': 1, 'current_zones': [], 'entered_zones': [], 'has_clip': False, 'has_snapshot': True, 'attributes': {}, 'current_attributes': [], 'pending_loitering': False}, 'type': 'end'}
2024-11-04 14:32:53,549 - __main__ - DEBUG - Getting snapshot for event: 1730727171.175817-t6tyxn, Crop: True
2024-11-04 14:32:53,549 - __main__ - DEBUG - event URL: http://192.168.111.10:5000/api/events/1730727171.175817-t6tyxn/snapshot.jpg
2024-11-04 14:32:53,627 - __main__ - ERROR - Error getting snapshot: 422
`

The works fine from my computer. Any idea why I receive error code 422?

**Comment by lubomir-moric** [comment]
Same here after updating to Frigate 0.15.0 Beta 1. Before that in v0.14.1 this issue was only occasional, as Frigate delaying storing the snapshot into DB sometimes.

**Comment by thomas-hass** [comment]
I see. That explains why the picture is available for me over during manual checks

**Comment by thomas-hass** [comment]
Gah, no matter what I try Frigate always returns 422.

**Comment by kyle4269** [comment]
0.16 will have LPR built in.

**Comment by thomas-hass** [comment]
Cool :) 

    response = requests.get(snapshot_url, params={ "crop": 1, "h": 500, "quality": 95 })

Now I get the image.

When will 0.16 be released?

**Comment by kyle4269** [comment]
> Cool :) 
> 
> 
> 
>     response = requests.get(snapshot_url, params={ "crop": 1, "h": 500, "quality": 95 })
> 
> 
> 
> Now I get the image.
> 
> 
> 
> When will 0.16 be released?

Yeah the problem is with the true or false in that request.  It needs to be 0 or 1. But I'm not sure, I just follow the PRs pretty closely and saw it.

**Comment by ljmerza** [comment]
should be fixed on latest relase
